### PR TITLE
chore: remove stale feature flags

### LINKED ISSUE #1590: Chore: Remove Editable Telegraf Configurations Feature Flag
We need to remove the Editable Telegraf Configurations feature flag from within ConfigCat and our code as the flag has not been touched for almost a year. 

**Acceptance Criteria**

* All instances of Editable Telegraf Configurations feature flag are removed from the UI Repo
* Editable Telegraf Configurations feature flag is removed from ConfigCat

### PR DESCRIPTION
**Pull request by Asalem1**
Closes #1590

### CODE REVIEW

File: src/variables/utils/hydrateVars.ts [hunk 1]
```diff
@@ -113,47 +112,6 @@ const collectAncestors = (
...
-  In the example below, we have 3 variables `a`, `b`, `c` where `a` is a parent to `b`, which a parent to `c`:
-
-  a --> b --> c
-
-  If `a`, `b`, or `c` are passed into this function, we should expect that the output for each should be:
-
-  `c` with a reference to its parent `b`, which should have a reference to its parent `a`
-*/
-
-const postOrderDFS = (
```
Comment (Asalem1) [inline src/variables/utils/hydrateVars.ts:129]: RIP hard work

**Review by Asalem1 — COMMENTED** [review]
RIP

File: src/telegrafs/components/TelegrafConfig.tsx [hunk 2]
```diff
@@ -36,7 +33,7 @@ export class TelegrafConfig extends PureComponent<Props> {
         <Editor
           script={config}
           onChangeScript={onChangeConfig}
-          readOnly={isFlagEnabled('editTelegrafs') === false}
+          readOnly={false}
```
Comment (drdelambre) [inline src/telegrafs/components/TelegrafConfig.tsx:36]: i think this should be true

File: src/telegrafs/components/TelegrafConfigOverlay.tsx [hunk 3]
```diff
@@ -15,14 +14,10 @@ import {ResourceType} from 'src/types'
...
   const {onClose} = useContext(OverlayContext)
-  const titleVerb = isFlagEnabled('editTelegrafs') ? 'Edit ' : ''
 
   return (
     <Overlay.Container maxWidth={1200} testID="telegraf-overlay">
-      <Overlay.Header
-        title={`${titleVerb}Telegraf Configuration`}
-        onDismiss={onClose}
-      />
+      <Overlay.Header title="Edit Telegraf Configuration" onDismiss={onClose} />
```
Comment (drdelambre) [inline src/telegrafs/components/TelegrafConfigOverlay.tsx:20]: this should be `Telegraf Configuration`

Comment (Asalem1) [reply src/telegrafs/components/TelegrafConfigOverlay.tsx:20]: but it's set to true currently

**Review by drdelambre — APPROVED** [review]
i think you're supposed to flip those conditions

File: src/telegrafs/components/TelegrafConfigOverlay.tsx (hunk 3 above)
Comment (drdelambre) [reply src/telegrafs/components/TelegrafConfigOverlay.tsx:20]: oh, nm

File: src/telegrafs/components/TelegrafConfig.tsx (hunk 2 above)
Comment (Asalem1) [reply src/telegrafs/components/TelegrafConfig.tsx:36]: but `true === false` will return `false`

**Comment by Asalem1** [comment]
> i think you're supposed to flip those conditions

I don't think so

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1597)

**Pull request merged**